behat.yml.dist

```
@@ -33,7 +33,7 @@ default:
               logo: "a.ecl-site-header-core__logo-link .ecl-site-header-core__logo-image, a.ecl-site-header-standardised__logo-link .ecl-site-header-standardised__logo-image"
               footer logo: "a.ecl-footer-core__logo-link .ecl-footer-core__logo-image-desktop, a.ecl-footer-standardised__logo-link .ecl-footer-standardised__logo-image-desktop"
               media container: ".ecl-media-container"
-              page header: ".ecl-page-header-core"
+              page header: ".ecl-page-header"
```

On the `ec` component with `core` branding we have an additional class, so we should target the page header like this:
```suggestion
              page header: ".ecl-page-header, .ecl-page-header--negative"
```

---

It is a selector to find the header. 
In your example "".ecl-page-header, .ecl-page-header--negative" browser will always use ".ecl-page-header" since both headers have this selector. Second selector is redundant. 
The type of the header is tested in tests/src/Kernel/Patterns/PageHeaderRenderingTest.php.